js/index.js

```diff
@@ -73,7 +74,7 @@ app.controller("IndexController", ["$scope", "$http", 'leafletData', function($s
   }
 
   function postSuccess(response) {
-    $scope.counter++;
+    $scope.counter = response.data.length;
```

Comment:
It looks like the loadTodos method is called right after this, and you also added this code to the loadTodos method, so now it is being run twice, right?